x-pack/plugins/secops/public/components/page/hosts/host_summary/index.tsx

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiDescriptionList,
+  EuiDescriptionListTitle,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiPanel,
+  EuiTitle,
+  EuiToolTip,
+} from '@elastic/eui';
+import { FormattedRelative } from '@kbn/i18n/react';
+import { get, getOr, isArray } from 'lodash/fp';
+import React from 'react';
+import { pure } from 'recompose';
+import { HostItem, HostsEdges } from 'x-pack/plugins/secops/server/graphql/types';
+import { DragEffects, DraggableWrapper } from '../../../drag_and_drop/draggable_wrapper';
```

Comment:
I usually add an extra line between regular imports and dotted local ones. We haven't added the linter rule for it yet ... But typically that's what I do FWIW. Not changes requested. Just mentioning.

Reply:
I like that convention as well -- I'll update.